### PR TITLE
Added instructions for adding redirects.

### PR DESCRIPTION
**Pull request by dblotsky**
- Added redirect types: `general`, `docs`, and `docs-global`.
- Added `deprecated.md` (will appear in other languages after translation).

### CODE REVIEW

**Comment by dblotsky** [comment]
@sarangan12 @riknoll @nikhilkh @purplecabbage @stevengill please review when you have a moment!

**Comment by riknoll** [comment]
LGTM, but I'd add more to the deprecated.html page. Just saying that the content is "deprecated" is sort of unclear. Instead, say something along the lines of "This page has been deprecated and does not exist in the 6.x version of the documentation. Click here to go to the 6.x overview page..."

**Comment by dblotsky** [comment]
@riknoll Updated.

**Comment by dblotsky** [comment]
@stevengill ping.

**Comment by dblotsky** [comment]
@riknoll any feedback on the new deprecated page?

**Comment by riknoll** [comment]
@dblotsky reviewing now

**Comment by purplecabbage** [comment]
lgtm!


@purplecabbage
risingj.com

**Comment by riknoll** [comment]
Minor nitpick: I would mention on the deprecated page that 6.0.0 is the latest version of the docs. It's not super clear where you got that version from, especially when you offer a redirect to "latest" below it. Maybe rephrase it as "This content is deprecated in the latest documentation (version 6.0.0)" or something like that. Makes it clear that those two versions are the same. Rest of the pr LGTM.

**Comment by stevengill** [comment]
I'm looking at this right now

**Comment by stevengill** [comment]
LGTM

**Comment by dblotsky** [comment]
@riknoll: updated.

**Comment by riknoll** [comment]
LGTM
